chpl/chpl-service/src/main/java/gov/healthit/chpl/attestation/dao/AttestationDAO.java

```diff
@@ -8,9 +8,11 @@
 
 import gov.healthit.chpl.attestation.domain.Attestation;
 import gov.healthit.chpl.attestation.domain.AttestationPeriod;
+import gov.healthit.chpl.attestation.domain.AttestationPeriodDeveloperException;
 import gov.healthit.chpl.attestation.domain.AttestationSubmittedResponse;
 import gov.healthit.chpl.attestation.domain.DeveloperAttestationSubmission;
 import gov.healthit.chpl.attestation.entity.AttestationEntity;
+import gov.healthit.chpl.attestation.entity.AttestationPeriodDeveloperExceptionEntity;
```

Comment:
When I saw this I was thinking it was a new type of Exception class and then realized it was not. I wish there was something else we could call this (at least in the code) besides an "exception" since that has a lot of meaning in Java. Anomaly, Variation, Deviation? Just wanted to mention it. At this point it would be a lot of written code to change if there even was another appropriate term.